<launch>
    <!-- Arguments -->
    <arg name="node_name" default="segmenter_ros" />

    <!-- Configurations -->
    <arg name="configs" default="$(find segmenter_ros)/config/params.yaml" />

    <!-- Nodes -->
    <node name="$(arg node_name)" pkg="segmenter_ros" type="segmenter.py" output="screen" >
        <rosparam command="load" file="$(arg configs)" />
    </node>
</launch>